.travis.yml

```diff
@@ -1,27 +1,25 @@
+sudo: required
+dist: trusty
 language: ruby
+before_install: gem install -N bundler -v '~> 1.11'
 cache: bundler
-sudo: false
 rvm:
-- 2.0.0
-- 2.1
-- 2.2
-- 2.3.0
-- jruby
+  - 2.1
+  - 2.2
+  - 2.3
+  - jruby-9.1.5.0
```

Comment:
JRuby 9.1.5.0 is a bugfix release of 9.1.X.Y. It is like MRI 2.3.1 vs 2.3.0. (note: we now specify 2.3 instead of 2.3.0, so it takes whatever 2.3.Y happens to be).